### PR TITLE
Add github info to /tfm

### PR DESCRIPTION
**Pull request by Scuph**
Just adding on the plugin's git.

### CODE REVIEW

File: src/me/StevenLawson/TotalFreedomMod/Commands/Command_tfm.java [hunk 1]
```diff
@@ -65,6 +65,7 @@ public boolean run(CommandSender sender, Player sender_p, Command cmd, String co
                 TotalFreedomMod.buildCreator), ChatColor.GOLD);
         playerMsg("Running on " + TFM_ConfigEntry.SERVER_NAME.getString() + ".", ChatColor.GOLD);
         playerMsg("Created by Madgeek1450 and Prozza.", ChatColor.GOLD);
+        playerMsg("If you wish to contribute to the development of this plugin, go to " + ChatColor.AQUA + "http://github.com/TotalFreedom/TotalFreedomMod.", ChatColor.GREEN);
```
Comment (JeromSar) [inline src/me/StevenLawson/TotalFreedomMod/Commands/Command_tfm.java]: How about? This plugin is opensource: https://github.com/TotalFreedom/TotalFreedomMod

**Comment by TheRevalation** [comment]
@JeromSar Done.

**Comment by OxLemonxO** [comment]
Change Plugin to plugin?

**Comment by JeromSar** [comment]
@Scuph See @OxLemonxO's suggestion :)

**Comment by JeromSar** [comment]
@Scuph Closing due to inactivity.

**Comment by Scuph** [comment]
@JeromSar Can you reopen this? I made all the changes.

**Comment by Wild1145** [comment]
@Scuph Just open a new PR.

**Pull request closed without merging**